backend/docker-compose.test-backend.yml

```diff
@@ -37,6 +43,20 @@ services:
             SUPERVISOR_HOST: ${SUPERVISOR_HOST}
             SUPERVISOR_PORT: ${SUPERVISOR_PORT}
             GUI_ENABLED: "false"
+            BULK_TIMER: 0.1
+            COVERALLS_REPO_TOKEN: coveralls_token_here
+            GIT_BRANCH: ${GIT_BRANCH}
```

Comment:
will all these variables be collected via .env or automatically set?

Reply:
no these are provided from SemaphoreCI